test/cmd_line/redo.test.ts

```diff
@@ -0,0 +1,23 @@
+import { getAndUpdateModeHandler } from '../../extension';
+import { ExCommandLine } from '../../src/cmd_line/commandLine';
+import { ModeHandler } from '../../src/mode/modeHandler';
+import { assertEqualLines, cleanUpWorkspace, setupWorkspace } from '../testUtils';
+
+suite('Redo command', () => {
+  let modeHandler: ModeHandler;
+
+  setup(async () => {
+    await setupWorkspace();
+    modeHandler = (await getAndUpdateModeHandler())!;
+  });
+
+  teardown(cleanUpWorkspace);
+
+  test('redoes last undoed action after insert mode', async () => {
+    await modeHandler.handleMultipleKeyEvents(['i', 'a', '<Esc>']);
```

Comment:
Test fails; I think because in an empty file `ia<Esc>ib<Esc>` results in `ba`, not `ab`